documents/WCA Bylaws.md

```diff
@@ -59,27 +97,30 @@ The number of authorized directors of the WCA shall be not less than three (3) n
 ### 3.3 Qualification of Directors
 No more than forty-nine percent (49%) of the directors serving on the Board may be interested persons, as defined in Section 5227 of the Nonprofit Corporation Law. However, any violation of this Section 3.3 shall not affect the validity or enforceability of any transaction entered into by the WCA.
 
-### 3.4 Election and Term of Office
+### 3.4 Eligibility of Directors
+To be eligible as a director of the WCA candidates must be at least 18 years of age and be a voting member of the WCA.
```

Comment:
```suggestion
To be eligible as a director of the WCA candidates must be at least 18 years of age and be an individual voting member of the WCA.
```

I know that only individuals are currently voting members, but e.g. if regional organisations gained voting rights then it would be useful clarifying it and tying eligibility to the phrasing used in section 2.